CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-vc-delivery ChangeLog
 
+## 3.1.0 - 2023-07-TBD
```

Comment:
Should I mark this as a breaking change? Given that the deployments will need to include a similar configuration.

Reply:
I don't think this needs to be breaking; it was a mistake not to have it and it didn't function without it. So there was already a requirement for it.

Reply:
Also, it's already been setup for deployments: https://github.com/digitalbazaar/terraform-gcp-veres-exchanger/blob/main/config.tftpl#L7